chaosk8s/pod/actions.py

```diff
@@ -46,21 +47,111 @@ def terminate_pods(label_selector: str = None, name_pattern: str = None,
     be used as the grace period (in seconds) to terminate the pods.
     Otherwise, the default pod's grace period will be used.
     """
+
```

Comment:
I'm confused about the next lines. Have they changed?

Reply:
Well I have added the function exec_in_pod which has the same logic for selecting the affected pods. So in order to avoid code duplication, I have isolated the pods selection logic in its own function, which is now called by  the two functions terminate_pod and exec_in_pod.